Tools/autotest/arducopter.py

```diff
@@ -11559,6 +11559,115 @@ def GripperReleaseOnThrustLoss(self):
         self.context_pop()
         self.reboot_sitl()
 
+    def CameraInformationFromJSON(self):
+        '''Tests that we can send CAMERA_INFORMATION message defined in JSON'''
+
+        json_filename = "camera_information.json"
+
+        # We use an example JSON file for this test...
+        example_json = os.path.realpath(os.path.join(testdir, "../../libraries/AP_Camera/json/", json_filename))
```

Comment:
with no try/except block around this stuff we may end up leaving stuff lying around that we shouldn't.

We fix this with scripting with `install_test_script_context` and friends.

I wonder if we need to generalise that stuff allow copying/deletion of arbitrary files.